ygen/protogen.go

```diff
@@ -183,6 +184,11 @@ import "{{ .YwrapperPath }}/ywrapper.proto";
 import "{{ .YextPath }}/yext.proto";
 {{- range $importedProto := .Imports }}
 import "{{ $importedProto }}";
+{{- end -}}
+
+{{- if ne .GoPackageName "" }}
```

Comment:
nit: `{{- if .GoPackageName }}` is sufficient.

Reply:
Thanks, fixed.